### PR TITLE
translate pods.md  and init-containers.md for branch release-1.16

### PR DESCRIPTION
**Pull request by liufangwai**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/concepts/workloads/pods/init-containers.md [hunk 1]
```diff
@@ -12,63 +12,119 @@ content_template: templates/concept
 
 {{% capture overview %}}
 
+<!--
+This page provides an overview of init containers: specialized containers that run before app containers in a {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+Init containers can contain utilities or setup scripts not present in an app image.
+-->
+
 本页提供了 Init 容器的概览，它是一种专用的容器，在应用容器启动之前运行，并包括一些应用镜像中不存在的实用工具和安装脚本。
```
Comment (markthink) [inline content/zh/docs/concepts/workloads/pods/init-containers.md]: {{< glossary_tooltip text="Pod" term_id="pod" >}} 文件丢失

Comment (liufangwai) [reply content/zh/docs/concepts/workloads/pods/init-containers.md]: historical problem, fixed

**Pull request merged**